tap_rest_api_msdk/tap.py

```diff
@@ -304,7 +304,7 @@ class TapRestApiMsdk(Tap):
         th.Property(
             "next_page_token_path",
             th.StringType,
-            default="$.next_page",
+            default=None,
             required=False,
             description="a jsonpath string representing the path to the 'next page' "
             "token. Defaults to `$.next_page`",
```

Comment:
Agreed. For the default paginator which is the `jsonpath_paginator` it will default to `$.next_page`. For all other paginators it defaults to None. I will make this more explicit in the documentation.